bool is_enable
int robot_status
bool is_connected
